scripts/AMDC_Logger.py

```diff
@@ -265,13 +258,49 @@ def load(self, file):
         out = pd.read_csv(p, comment = '#', index_col = 't')
         
         return out
+    
+    def _create_log_var(self, name, samples_per_sec, var_type, manual_index = None):
+        
+        memory_addr = int(self.mapfile.address(name), 0)
```

Comment:
Should this be `int(foo, 16)` since it is parsing a hex number? Or does `0` have it automatically figure it out?

Reply:
I'm not sure. You wrote that code. I just copied it into this new function

Reply:
Haha excellent, I'm now questioning my own code :)

Reply:
I mean it works haha

Reply:
Ah, from the docs:

> Base 0 means to interpret exactly as a code literal, so that the actual base is 2, 8, 10, or 16, and so that int('010', 0) is not legal, while int('010') is, as well as int('010', 8).

I guess `0` is a fine choice here! We can resolve this comment.

Reply:
^^ This is only true since the mapfile output is of the form `"0x01238124"` ... if it didn't have `0x` prefix, it wouldn't work.